src/controllers/TxInfoController.js

```diff
@@ -29,4 +27,44 @@ export class TxInfoController {
         throw new Error(await resp.text());
     }
   }
+
+  async txHistory(address, limit = 100) {
+    const url = new URL(
+      `transactions/address/${address}/limit/${limit}`,
+      this.getNode()
+    ).toString();
+
+    let resp = await fetch(url, {
+      headers: { accept: 'application/json; large-significand-format=string' },
+    });
+
+    switch (resp.status) {
+      case 200:
+        return resp.json().then(arr => (Array.isArray(arr) && arr[0]) || []);
```

Comment:
I doubt this `Array.isArray(...)` check is needed. Wouldn't this be enough?

```js
.then(arr => arr[0])
```

Reply:
I missed that node returns `[ [ ] ]` if there are no records. Of course, that will be enough.